MiniTwitch.Helix/Responses/ConduitShard.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Text.Json.Serialization;
+using MiniTwitch.Helix.Enums;
+using MiniTwitch.Helix.Internal.Json;
+
+namespace MiniTwitch.Helix.Responses;
+
+public record ConduitShard(
```

Comment:
This should be nested under ConduitsShards, see Responses/Chatters.cs as an example

Reply:
Fixed in 2e4a6b9ae27f337aade7ceeb56fddef351ab1c5b